js/sorting.js

```diff
@@ -0,0 +1,51 @@
+const Filter = {
+	DEFAULT: 'filter-default',
+	RANDOM: 'filter-random',
+	POPULAR: 'filter-discussed'
+};
+const sortingElement = document.querySelector('.img-filters');
+
+let currentFilter = Filter.DEFAULT;
+let photos = [];
+
+const sortInRandom = () => Math.random() - 0.5;
+
+const sortInPopular = (photo1, photo2) =>
+	photo2.comments.length - photo1.comments.length;
+
+const getSortingPhotos = () => {
+	switch (currentFilter) {
+	case Filter.RANDOM:
+		return [...photos].sort(sortInRandom);
+	case Filter.POPULAR:
+		return [...photos].sort(sortInPopular);
+	default:
+		return [...photos];
```

Comment:
ну конкретно тут необязательно 
```suggestion
		return photos;
```